### PR TITLE
feat(AR-1513): Playground tests for subgraph

### PR DESCRIPTION
**Pull request by Paul-okpor**

### CODE REVIEW

File: nodejs/graphql/handlers/subgraph.handler.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+const query = JSON.stringify({query: '{ transfers(first: 5) { id sender recipient amount }'});
+
+
+
+class SubgraphHandler {
+    getTransfers = async (isMainnet)=>{
+
+        try {
+            const response = await axios.post(`${urlHandler.getSubgraphURL(isMainnet)}/aba76682a6nb4K6bDDN6aa71N1c4NN8a/erc20-transfers-quickstart/`, {
```
Comment (syed-bcw) [inline nodejs/graphql/handlers/subgraph.handler.js:13]: `ARKHIA_API_KEY` hardcoded maybe

Comment (daniel-bcw) [reply nodejs/graphql/handlers/subgraph.handler.js:13]: Yes lets definitely remove that end and get it from the .env

Comment (daniel-bcw) [reply nodejs/graphql/handlers/subgraph.handler.js:13]: @Paul-okpor i think u can put this in the class RestApiHandler since its a post ?

Comment (Paul-okpor) [reply nodejs/graphql/handlers/subgraph.handler.js]: I think its better here because i will be expanding the subgraph

**Review by syed-bcw — APPROVED** [review]

**Pull request merged**